### DISCUSSION TITLE
Adressblatt

### DISCUSSION THREAD

**Discussion by honel** [openjverein/jverein#502] [Ideas]
Ich möchte gerne aus dem Mitgliederbereich ein Adressblatt (gerne auch mit wählbaren standardisierten Anschreiben) ausdrucken können. Ist es möglich so eine Funktion zu integrieren?

Vielen Dank!!

**Comment by lenilsas** [comment]
Du Kannst ein Freies Formular erstellen unter Einstellungen->Mitglieder->Formulare. Erklärung dazu in der OnineDoku

**Comment by honel** [comment]
Danke für da schnelle Feedback. Das wäre ein Workaround für mein Problem.

(Es gab mal die Option mit der Spendenbescheinigung ein Adressblatt auszudrucken. Diese nützliche Option gibt es aber in der aktuellen Version nicht mehr. Also werde ich es über das "Freie Formular" einrichten.)

**Comment by lenilsas** [comment]
Bei Spendenbescheinigungen gibt es das nach wie vor, jedoch nur noch bei Druck&Mail->Spendenbescheinigung

**Reply by honel** [reply]
Danke.
